web/src/components/pages/dashboard/challenge/leaderboard-card.tsx

```diff
@@ -41,7 +41,7 @@ export default function LeaderboardCard({
   const leaderboardRef = useRef(null);
   return (
     <StatsCard
-      key="leaderboard"
```

Comment:
`key` is not supported as a prop, it's used interally in React for rendering things like lists. Renamed it to `id` for this reason.